src/qililab/digital/circuit_transpiler.py

```diff
@@ -89,20 +101,20 @@ def transpile_circuits(
             platform = build_platform(runcard="<path_to_runcard>")
 
             # Create transpiler:
-            transpiler = CircuitTranspiler(platform)
+            transpiler = CircuitTranspiler(platform)  # Or CircuitTranspiler(platform.digital_compilation_settings)
```

Comment:
change this

Reply:
okei, I'll leave only the first, and with the docstring it should be understandable 👌